include/seahorn/UfoOpMemSem.hh

```diff
@@ -0,0 +1,90 @@
+#pragma once
+// This is a copy of UfoOpSem for prototyping purposes
```

Comment:
`MemUfoOpSem` would be a better name. `OpSem` stands for *operational semantics*, so puting `Mem` in between does not work

Reply:
Can this be refactored into a base template and two implementations so that code that is common between `UfoOpSem` and `UofOpMemSem` is shared

Reply:
Done